svc-systems/chassis/update.go

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package chassis
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/ODIM-Project/ODIM/lib-utilities/common"
+	"github.com/ODIM-Project/ODIM/lib-utilities/errors"
+	"github.com/ODIM-Project/ODIM/lib-utilities/proto/chassis"
+	"github.com/ODIM-Project/ODIM/lib-utilities/response"
+	"github.com/ODIM-Project/ODIM/svc-systems/plugin"
+)
+
+func (h *Update) Handle(req *chassis.UpdateChassisRequest) response.RPC {
+	pc, e := h.createPluginClient("URP*")
```

Comment:
IMHO plugin id shouldn't be hardcoded, plugin info should be fetched using `ManagedBy` info, and then the same need to be used

Reply:
You are right your proposal makes sense in this case(update/PATCH). Unfortunately, plugin name has to be hardcoded in case of GET/POST requests. So for keeping implementation consistent across different calls(GET/POST/PATCH) I would prefer keep plugin name hardcoded.

Reply:
Pluginid can be retrieved from database using ManagedBy attribute that is available at the time of sending PUT/POST. Whereas GET/DELETE/PATCH we can get to the plugin info from the database. I think we don't have a need to hardcode the plugin id.

Please fix this.

Reply:
Understand that GET cannot managedby currently from DB as explained by Robert. So we can leave this as is for now and approve. 

Will revisit to see how we can fix this in a later release